### DISCUSSION TITLE
POST request to API using Form Input

### DISCUSSION THREAD

**Discussion by ashwani-cm** [Budibase/budibase#7943] [Help] [answered]
Folks, I want to make a POST request to a REST API using the Form input data, can you please help me with this, I'm unable to figure out how can we send the form data as request body

Will be obliged for your help
Thanks

**Comment by melohagan** [comment]
Hi @ashwani-cm 

Have you looked at REST bindings docs? https://docs.budibase.com/docs/rest-bindings#supplying-data-to-a-binding

**Comment by jlangree** [comment, marked as answer]
I haven't figured out a way to include JSON data as a binding though. Either as a `JSON.stringify()`'d version or as object data. It seems like including the bindings in the POST body is only possible as a string. However, creating an action on a button to send form data to a REST binding as stringified JSON fails. See me other post about that here: #8496 

EDIT: Never mind. You can send the JSON object _not as a string_ by not putting the binding inside quotes in the post body.

eg:
```json
{
  "data" {{ yourJsonObjBinding }}
}
```
